install/gcp/deployment_manager/istio-cluster.jinja

```diff
@@ -97,7 +98,31 @@ resources:
           {% endif %}
 
           {% if  properties['enableAutomaticSidecarInjection'] %}
-            kubectl apply -f install/kubernetes/istio-initializer.yaml
+            {% if properties['installIstioRelease'] == "0.5.0" %}
+              # Setup automatic istio sidecar injection for the default namespace.
+              # Note that the original 0.5.0-tagged version of this script had
+	      # a timing bug; here we pull one commit later.
+              curl -s https://raw.githubusercontent.com/istio/istio/41203341818c4dada2ea5385cfedc7859c01e957/install/kubernetes/webhook-create-signed-cert.sh -o ./install/kubernetes/webhook-create-signed-cert.sh && chmod +x ./install/kubernetes/webhook-create-signed-cert.sh
+              curl -s https://raw.githubusercontent.com/istio/istio/0.5.0/install/kubernetes/webhook-patch-ca-bundle.sh -o ./install/kubernetes/webhook-patch-ca-bundle.sh && chmod +x ./install/kubernetes/webhook-patch-ca-bundle.sh
+
+              ./install/kubernetes/webhook-create-signed-cert.sh \
+                  --service istio-sidecar-injector \
+                      --namespace istio-system \
+                      --secret sidecar-injector-certs
+              kubectl apply -f install/kubernetes/istio-sidecar-injector-configmap-release.yaml
+              cat install/kubernetes/istio-sidecar-injector.yaml | \
+                  ./install/kubernetes/webhook-patch-ca-bundle.sh > \
+                  install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml
+              kubectl apply -f install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml
+              kubectl label namespace default istio-injection=enabled
```

Comment:
UX question - this only enables injection in the `default` namespace. Do we want it enabled cluster-wide? Or should instructions be updated to educate user that namespaces must be explicitly labeled to be subject to automatic injection?

Reply:
sorry, i added in the response in separately below re ```default``` namespace.  We're creating the a new GKE cluster from scratch everythime so its fine to use the default.  I've got a PR i want to submit next week for the docs where i'll state that the script is only enabling auto injection for defaults